tools/pipelines-tasks/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Windows Store app package directories and files

```

Comment:
Not sure that we need this file

Reply:
I removed everything that wasn't needed from it. This file applies only to this directory, and there are a couple of things that we want to ignore only on this project. For example, this ignores all .js files because the project uses TypeScript and compiles to JavaScript, but we probably don't want to ignore JS on the whole repo.